image_api/init.sql

```diff
@@ -0,0 +1,10 @@
+create  extension if not exists  "uuid-ossp";
```

Comment:
you do not need uuid extension just to mention uuid type. you need it to generate uuid value